bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckablePullRequest.java

```diff
@@ -143,6 +143,16 @@ static List<Review> filterActiveReviews(List<Review> allReviews, String targetRe
                 if ((prevReviewCorrectTarget && reviewCorrectTarget && reviewNewer)
                         || (!prevReviewCorrectTarget && !reviewCorrectTarget && reviewNewer)
                         || (!prevReviewCorrectTarget && reviewCorrectTarget)) {
+                    // If current review is just a comment, the verdict should inherit from the previous review
+                    if (review.verdict().equals(Review.Verdict.NONE)) {
+                        review = new Review(review.createdAt(),
+                                review.reviewer(),
+                                prevReview.verdict(),
+                                review.hash().isPresent() ? review.hash().get() : null,
+                                review.id(),
+                                review.body().isPresent() ? review.body().get() : null,
+                                review.targetRef());
+                    }
```

Comment:
I don't whether it is a good idea to create a new review here.

If we don't want to create a new review, we should make `verdict` not final and add a set method in Review.

Reply:
I don't think we should create a new `Review` here. I _think_ that the way to achieve the desired behavior is to ignore reviews where the verdict isn't `APPROVED` or `DISAPPROVED`. A `Review` with verdict `NONE` is essentially pointless when considering active reviews.

Reply:
> I don't think we should create a new `Review` here. I _think_ that the way to achieve the desired behavior is to ignore reviews where the verdict isn't `APPROVED` or `DISAPPROVED`. A `Review` with verdict `NONE` is essentially pointless when considering active reviews.

Ah, exactly! Thanks for the suggestion! I will change it.